src/db/collections/milestone.rs

```diff
@@ -42,6 +42,8 @@ pub struct MilestoneDocument {
     at: MilestoneIndexTimestamp,
     /// The milestone's payload.
     payload: MilestonePayload,
+    /// The milestone stats.
+    stats: MilestoneStats,
```

Comment:
Can we create a new collection for this please, like `stardust_analytics_milestones`?